app/src/openxr/cpp/OpenXRLayers.cpp

```diff
@@ -242,4 +242,40 @@ OpenXRLayerEquirect::GetNextStructureInChain() const {
 }
 #endif
 
+// OpenXRLayerPassthrough;
+
+OpenXRLayerPassthroughPtr
+OpenXRLayerPassthrough::Create(const VRLayerPassthroughPtr& aLayer, XrPassthroughFB passthrough) {
```

Comment:
I'm a bit worried about having a dependency with the FB extension here. Other systems use different ones, so at some point we need a factory that creates the proper instance. Maybe we can have this as a first version, but I hope this does not tie our hands for future changes.